src/basic/missing_syscalls.py

```diff
@@ -21,8 +23,10 @@
     'pidfd_send_signal',
     'pkey_mprotect',
     'quotactl_fd',
+    'removexattrat',
     'renameat2',
     'setns',
+    'setxattrat',
     'statx',
```

Comment:
might make sense to add these to src/share/seccomp-util.c to the syscall_filter_sets[] array for @file-system

Reply:
Done by fb35d775a already?